src/applications/appeals/10182/components/VeteranInformation.jsx

```diff
@@ -33,27 +34,38 @@ const VeteranInformation = ({ profile = {}, veteran = {} }) => {
         Confirm the personal information we have on file for you.
       </h3>
       <div className="blue-bar-block">
-        <strong className="name">
+        <strong className="name dd-privacy-mask">
```

Comment:
I'm being a bit paranoid here, but let's make this one hidden so no one can infer identity by number of characters in name. Some names are uncharacteristically short or long.

Reply:
I also hid the gender - masking would still make the answer obvious.